### PR TITLE
476 ee leak

### PR DESCRIPTION
**Pull request by tonysnowboardunderthebridge**
Hi @marcopiraccini 

updates for removeListeners from electron when using useEffect

Resolve https://github.com/platformatic/meraki/issues/476

### CODE REVIEW

**Review by marcopiraccini — CHANGES_REQUESTED** [review]
If you remove ALL listener, you don't need to pass the listener, check: https://nodejs.org/docs/latest/api/events.html#emitterremovealllistenerseventname

File: src/renderer/src/api.js [hunk 1]
```diff
@@ -53,6 +55,7 @@ export const callApiStartLogs = async (id) => {
   return await window.api.startLogs(id)
 }
 export const getAppLogs = callback => window.api.onAppLog(callback)
+export const removeAppLogs = callback => window.api.removeAppLog(callback)
```
Comment (marcopiraccini) [inline src/renderer/src/api.js:58]: I don't think you need to pass the listener

Comment (tonysnowboardunderthebridge) [reply src/renderer/src/api.js]: Done!

**Review by marcopiraccini — APPROVED** [review]
lgtm

**Pull request merged**